html_css/v2/main-course/positioning.md

```diff
@@ -0,0 +1,38 @@
+### Positioning
+
+You have learned various ways how to position elements on the screen and even though the methods you know so far are sufficient in most cases, these CSS rules are still very powerful and will certainly help you step up your CSS game!
+
+### Learning Outcomes
+ - You'll learn how to use absolute positioning.
+ - You'll learn how to use fixed positioning.
+ - You'll learn how to use sticky positioning.
+ - You'll know the difference between each property and how to combine them.
+
+### Static and Relative Positioning
+The difference between static and relative is fairly simple. Static is the default position of every element, and properties _top_, _right_, _bottom_, and _left_ have no affect on statically positioned elements. Relative on the other hand is pretty much the same as static, but properties  _top_, _right_, _bottom_, and _left_ affect the specific element relative to itself.
+
+### Absolute Positioning
+Absolute positioning allows you to _absolutely_ position elements on the screen. Absolutely positioned elements will be relative to the next absolute or relative parent. Absolutely positioned elements are also removed from the normal flow of the document. To put it in simple terms: elements that are removed from the normal flow of the document don't affect other elements and are also not affected by other elements. Using absolute positioning allows you to position elements anywhere on the screen using  _top_, _right_, _bottom_, and _left_ properties. This property is really useful when you want to position something at an exact point on the screen, without disturbing any of the other elements.
+
+### Fixed Positioning 
+Fixed elements are also removed from the normal flow of the document and are positioned relative to the _viewport_. You basically use  _top_, _right_, _bottom_, and _left_ properties to position it, and it will stay there as the user scrolls. This is especially useful for things like navigation bars.
+
+### Sticky Positioning
+Sticky elements will act like normal elements until you scroll past them, then they start behaving like fixed elements. They are also not taken out of the normal flow of the document. It might sound confusing, but checking out the behavior of <a href="https://codepen.io/theanam/pen/MPLBYy">this</a> example might clear things up for you.
```

Comment:
useful for things like section-headings on lists or tables that can scroll off the screen.  for example it's nice to be able to see what 'category' you're looking at on a long scrolling list, even tho the header should have scrolled off the screen.